db/migrate/20230712140459_create_comments.rb

```diff
@@ -8,4 +8,4 @@ def change
       t.timestamps
     end
   end
-end
+end
```

Comment:
- [OPTIONAL] In this [schema](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png), the `User` has many `Comments` through the column `author_id`. You need to mirror this relationship by properly configuring the migration file that creates the Comment model. I would advise you to make your migration file look this way 😎 👍:

```ruby
create_table :comments do |t|
    t.timestamps
    t.text :text
    t.references :author, foreign_key: { to_table: 'users' }, null: false, index: true 
    t.references :post, foreign_key: true 
  end
```